src/docs/patterns/user-auth.mdx

```diff
@@ -225,7 +229,7 @@ Some applications, especially consumer-facing ones, can be authorized via third-
 -   [Facebook Login](http://developers.facebook.com/docs/facebook-login/)
 -   [Sign in with Microsoft](https://developer.microsoft.com/en-us/identity/add-sign-in-with-microsoft)
 
-## Forgot / Change Password
+## Forgot / Changing Password
```

Comment:
I would leave this as it was...with this change you are using two different verb forms (gerund & past participle)